tests/integrations/basic-fail/tests/actual_tests/wrong_diagnostic_code.rs

```diff
@@ -0,0 +1,9 @@
+#![warn(dead_code, unreachable_code)]
```

Comment:
Why is this test in basic-fail, and not somewhere where it'll get blessed?

Reply:
Is that not the correct spot for a test with an unmatched comment?

Reply:
I did not realize that's what you're testing. Please add a comment and move it to one of the bless directories. Also I don't think you're testing that from the diff I'm seeing, as it just looks like the test is missing its stderr output and is testing that a pass test in a fail directory will emit a message about the wrong exit code

Reply:
The first comment has the wrong code. The correct code for the first one is `dead_code`. The second `unreachable_code` comment is correct.

I'll add a comment to the test.

Reply:
Oh... maybe I'm not emitting annotation mismatch errors if there were stderr differences

Reply:
I can try to move it and see what happens.

Reply:
Moved it to `actual_tests_bless` which does give a `.stderr` file. Is this what you meant in the first place?